dockerfiles/build.include

```diff
@@ -0,0 +1,151 @@
+#!/bin/sh
```

Comment:
if it's because you need the files, we may have a pom.xml in dockerfiles/ as well to embed the sources